scripts/generate_core_config.sh

```diff
@@ -208,6 +208,14 @@ db-pass=$local_DATAFED_DATABASE_PASSWORD
 # with what is in the datafed-ws.cfg file
 client-id=$local_DATAFED_GLOBUS_APP_ID
 client-secret=$local_DATAFED_GLOBUS_APP_SECRET
+# Below sets the log level of the application
+# 0 - Critical
+# 1 - Error
+# 2 - Warning
+# 3 - Info  (Default)
+# 4 - Debug
+# 5 - Trace
+log-level=4
```

Comment:
Do we want the default to be 3 or 4?

```shell
3 - Info  (Default)
```

Reply:
Should be Info, you don't want people generating more logs than they need on a production system just because they didn't pay attention to the log settings.

Reply:
How should I address this for the CI? The image is built using this script and it is run in the entrypoint, so I'm not sure how to overwrite it.